slither.config.json

```diff
@@ -1,9 +1,10 @@
 {
   "detectors_to_exclude": "naming-convention,reentrancy-events,solc-version,timestamp",
-  "filter_paths": "(node_modules,test)",
+  "filter_paths": "(node_modules/,test/)",
   "solc_remaps": [
     "@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/",
     "@prb/math/=node_modules/@prb-math/",
+    "@sphinx-labs/contracts/=node_modules/@sphinx-labs/contracts/contracts/foundry",
```

Comment:
shouldn't we add "/" at the end?

Reply:
I guess? I'll add it.

Reply:
maybe this was the problem with Sphinx all along? :D

cc @smol-ninja